guides/release/tutorial/9.md

```diff
@@ -0,0 +1,152 @@
+
+Let's try it out by editing the index template:
+
+```handlebars {data-filename="app/templates/index.hbs" data-diff="-1,-2,+3,-7,+8"}
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <Jumbo>
+    <h2>Welcome to Super Rentals!</h2>
+    <p>We hope you find exactly what you're looking for in a place to stay.</p>
+    <LinkTo @route="about" class="button">About Us</LinkTo>
+  </Jumbo>
+</div>
+```
+
+After saving the changes, your page should automatically reload, and, _voilà_... nothing changed? Well, that's exactly what we wanted to happen this time! We successfully *[refactored](TODO: link to refactored)* our index template to use the `<Jumbo>` component, and everything still works as expected. All the tests are still passing!
+
+<!-- TODO: screenshot of running tests? -->
+
+Let's do the same for our other two pages as well.
+
+```handlebars {data-filename="app/templates/about.hbs" data-diff="-1,-2,+3,-11,+12"}
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <Jumbo>
+    <h1>About Super Rentals</h1>
+    <p>
+      The Super Rentals website is a delightful project created to explore Ember.
+      By building a property rental site, we can simultaneously imagine traveling AND building Ember applications.
+    </p>
+    <LinkTo @route="contact" class="button">Contact Us</LinkTo>
+  </Jumbo>
+</div>
+```
+
+```handlebars {data-filename="app/templates/contact.hbs" data-diff="-1,-2,+3,-19,+20"}
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <Jumbo>
+    <h2>Contact Us</h2>
+    <p>
+      Super Rentals Representatives would love to help you<br>
+      choose a destination or answer any questions you may have.
+    </p>
+    <address>
+      Super Rentals HQ
+      <p>
+        1212 Test Address Avenue<br>
+        Testington, OR 97233
+      </p>
+      <a href="tel:503.555.1212">+1 (503) 555-1212</a><br>
+      <a href="mailto:superrentalsrep@emberjs.com">superrentalsrep@emberjs.com</a>
+    </address>
+    <LinkTo @route="about" class="button">About</LinkTo>
+  </Jumbo>
+</div>
+```
+
+After saving, everything should look exactly the same as before, and all the tests should still pass. Very nice!
+
+<!-- TODO: screenshot of the page? -->
+
+While it may not save you a lot of characters in this case, it made the template slightly easier to read by *[encapsulating](TODO: link to encapsulating)* the implementation of the "jumbo" header into its own component, which allowed the reader to focus on things that are unique to just that page. Now, if we need to make a change to the header, we will just have to make it in a single place. Feel free to give that a try!
```

Comment:
```suggestion
While it may not save you a lot of characters in this case, [encapsulating](TODO: link to encapsulating)* the implementation of the "jumbo" header into its own component makes the template slightly easier to read, and it allows the reader to focus on things that are unique to just that page. Now, if we need to make a change to the header, we can make it in a single place. Feel free to give that a try!
```